addons/web_editor/static/src/js/backend/html_field.js

```diff
@@ -404,7 +404,9 @@ export class HtmlField extends Component {
     _isDirty() {
         const strippedPropValue = stripHistoryIds(String(this.props.value));
         const strippedEditingValue = stripHistoryIds(this.getEditingValue());
-        return !this.props.readonly && (strippedPropValue || '<p><br></p>') !== strippedEditingValue;
+        const domParser = new DOMParser();
+        const valueChange = domParser.parseFromString(strippedPropValue || '<p><br></p>', 'text/html').body.innerHTML !== domParser.parseFromString(strippedEditingValue, 'text/html').body.innerHTML;
+        return !this.props.readonly && valueChange;
```

Comment:
I think line 408 is too long and too hard to read. I suggest the following:
```js
const parsedPropValue = domParser.parseFromString(strippedPropValue || '<p><br></p>', 'text/html').body;
const parsedEditingValue = domParser.parseFromString(strippedEditingValue, 'text/html').body;
return !this.props.readonly && parsedPropValue.innerHTML !== parsedEditingValue.innerHTML;
```

Reply:
Thanksss!